src/EnergyPlus/ZoneTempPredictorCorrector.hh

```diff
@@ -226,6 +247,12 @@ namespace ZoneTempPredictorCorrector {
 	CalcZoneAirTempSetPoints();
 
 	void
+	CalculateMonthlyRunningAverageDryBulb( Array1D< Real64 > & runningAverageASH, Array1D< Real64 > & runningAverageCEN );
```

Comment:
@XuanLuoLBL In the other function, it is passing the arrays by value, which means it will make a copy of those arrays every time. You should pass them by const reference if you want them to not change, but still not make a new copy every time.

```cpp
CalculateAdaptiveComfortSetPointSchl( Array1D< Real64 > const & runningAverageASH, Array1D< Real64 > const & runningAverageCEN )
```